### ISSUE TITLE
showfull is broken

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalArithmetic.jl#205]
Or at least does not behave like we would like:

```jl
julia> showfull(stdout, x)
Interval(3.0, 4.0)
julia> showfull(x)
ERROR: MethodError: no method matching showfull(::Interval{Float64})
Closest candidates are:
  showfull(::IO, ::Interval{S}) where S at /Users/dpsanders/.julia/packages/IntervalArithmetic/iO2gf/src/display.jl:259
  showfull(::IO, ::DecoratedInterval{S}) where S at /Users/dpsanders/.julia/packages/IntervalArithmetic/iO2gf/src/display.jl:259
  showfull(::IO, ::IntervalBox) at /Users/dpsanders/.julia/packages/IntervalArithmetic/iO2gf/src/display.jl:265
Stacktrace:
 [1] top-level scope at none:0

julia> showfull(stdout, x)
Interval(3.0, 4.0)
```

**Comment by lbenet** [comment]
`showfull(x) = showfull(stdout, x)` solves the problem.

**Comment by dpsanders** [comment]
Fixed by #249.
